EIPS/EIP-draft-getLogs-by-block-hash.md

```diff
@@ -0,0 +1,50 @@
+
+By adding the ability to fetch logs by hash, the client can be guaranteed that if they get a result set, it is for the block in question.  If they get an error, then they can take appropriate action (e.g., rollback that block client-side and re-fetch latest).
+
+## Backwards Compatibility
+
+The only potential issue here is the `fromBlock` and `toBlock` fields.  It wouldn't make sense to include both the hash and the number so it seems like `fromBlock`/`toBlock` should be mutually exclusive with `blockHash`.
+
+## Test Cases
+
+`{ "jsonrpc": "2.0", "id": 1, "method": "eth_getLogs", params: [{"blockHash": "0xbl0cbl0cbl0cbl0cbl0cbl0cbl0cbl0cbl0cbl0cbl0cbl0cbl0cbl0cbl0cbl0c"}] }` should return all of the logs for the block with hash `0xbl0cbl0cbl0cbl0cbl0cbl0cbl0cbl0cbl0cbl0cbl0cbl0cbl0cbl0cbl0cbl0c`.  If a `topics` field is added to the filter options then a filtered set of logs for that block should be returned.  If no block exists with that hash then an error should be returned with a `code` of `-32602`, a `message` of `"Invalid params"` and a `data` of `"Block with hash 0xbl0cbl0cbl0cbl0cbl0cbl0cbl0cbl0cbl0cbl0cbl0cbl0cbl0cbl0cbl0cbl0c was not found."`.
+
+## Implementation
+
+- [ ] Geth
+- [ ] Parity
+- [ ] EthereumJ
+
+## Copyright
+
+Copyright and related rights waived via CC0.
```

Comment:
Please add a link to CC0.  https://creativecommons.org/publicdomain/zero/1.0/

Reply:
Fixed.